cli/init.go

```diff
@@ -70,13 +91,13 @@ var initCmd = &cobra.Command{
 
 		log.SuccessStatusEvent(os.Stdout, "Congratulations! You have initialized the stream function successfully.")
 		log.InfoStatusEvent(os.Stdout, "You can enjoy the YoMo Stream Function via the command: ")
-		log.InfoStatusEvent(os.Stdout, "\tStep 1: cd %s && yomo build", name)
-		log.InfoStatusEvent(os.Stdout, "\tStep 2: yomo run sfn.yomo")
+		log.InfoStatusEvent(os.Stdout, "\tcd %s && yomo run", name)
```

Comment:
Should the `yomo build` command be deprecated? Migrate the `wasi` flag to `yomo run` command..